operator/pkg/utils/useragent.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/stackrox/rox/pkg/images/defaults"
+	"github.com/stackrox/rox/pkg/version"
+	"k8s.io/client-go/rest"
+)
+
+// SetUserAgent returns a rest.Config configured to set the UserAgent header when sending HTTP requests to K8s API Server.
+func SetUserAgent(config *rest.Config) *rest.Config {
```

Comment:
What do you think about creating a custom config getter which delegates to `GetConfigOrDie`  to have a canonical location to read the configured rest config used by the operator.

```
func GetConfigOrDie() *rest.Config {
 cfg := ctrl.GetConfigOrDie()
 cfg.UserAgent = getUserAgent()
 return cfg
}
```

Reply:
Looks good, this way we don't have to change the calling code for config changes in the future. Done.